src/Measure.ts

```diff
@@ -6,6 +6,25 @@ import { AggregateFunctionType } from "./AggregateFunctionType";
 import { Organization } from "./Organization";
 import { BaseConfigurationTypes } from "./BaseConfigurationTypes";
 
+export type Code = {
+  name: string;
+  display: string;
+  version: string;
+  codeSystem: string;
+  codeSystemOid: string;
+  status: string;
+}
+export type CodeSystemVersion = { 
```

Comment:
can we not rename it to CodeSystem instead of CodeSystemVersion because wording of CodeSystemVersion is more confining it to version but not to entire Code System props